styles/design-settings/cardboard/parts/_titles-settings.scss

```diff
@@ -1,6 +1,6 @@
 @include add_settings((
     PageTitles: (
-        _selectors: ('[data-type="page"]:not(.introduction)'),
+        _selectors: ('[data-type="page"]:not(.introduction):not(.unit-opener)'),
```

Comment:
I needed to exclude this selector from Page Titles shape to prevent overwriting styles for unit page tile.